packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx

```diff
@@ -109,6 +109,6 @@ export class DetailsListBasicExample extends React.Component<{}, IDetailsListBas
   };
 
   private _onItemInvoked = (item: IDetailsListBasicExampleItem): void => {
-    alert(`Item invoked: ${item.name}`);
+    console.log(`Item invoked: ${item.name}`);
```

Comment:
do we need to update any docs that refer to this being an alert? i.e. is there text that says "double click to see an alert". I'm guessing no, but thought I would ask.

Reply:
done. :)